docs/running-tests/from-local-system.md

```diff
@@ -22,6 +22,10 @@ On Debian or Ubuntu:
 sudo apt-get install python python-pip virtualenv
 ```
 
+On other Linux distros, it is important to have a package that provides a
+`python` binary. On Fedora, for example, that means installing the
+`python-unversioned-command` package.
```

Comment:
It looks good. You can add this: On Ubuntu, it means installing the package ```python-is-python3```

Reply:
Done!